rdrf/rdrf/testing/behaviour/features/steps.py

```diff
@@ -1012,20 +1012,20 @@ def check_radio_selected(step, value, cde, no):
         "/label[contains(.,'%s')]"
         % (cde, value)
     )
-    innerHTML = find(xp).get_attribute("innerHTML")
+    innerhtml = find(xp).get_attribute("innerHTML")
     if no:
-        assert "checked=\"\"" not in innerHTML,\
+        assert "checked=\"\"" not in innerhtml,\
```

Comment:
repeating that ugly "checked=\"\"" is not nice

do as I said before and give  a semantic meaning to "checked=\"\"" by giving it a name in one place.

What is "checked=\"\""
?